react-accordion-component/dist/styles.css

```diff
@@ -0,0 +1,33 @@
+.one,
+.two,
+.three {
+  border: 3px solid black;
+  background: rgb(219, 218, 218);
+  color: black;
+  font-family: sans-serif;
+  width: 350px;
+  padding: 5px;
+  margin-bottom: 3px;
+}
+
+.container {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.text {
```

Comment:
Something a bit more descriptive would be a better class name, like: `accordion-description`